packages/react-tabster/tsconfig.json

```diff
@@ -1,17 +1,17 @@
 {
   "extends": "../../tsconfig.base.json",
+  "include": ["src"],
   "compilerOptions": {
     "target": "ES5",
-    "lib": ["ES2015", "DOM"],
+    "module": "CommonJS",
+    "lib": ["ES2015", "dom"],
```

Comment:
this package needs 2015 as well ?

Reply:
Unfortuntely yeah, since it recently started to depend on makeStyles